pkg/api/customization/clustertemplate/cluster_template.go

```diff
@@ -57,3 +68,61 @@ func (w Wrapper) LinkHandler(apiContext *types.APIContext, next types.RequestHan
 	}
 	return nil
 }
+
+func (w Wrapper) ClusterTemplateRevisionsActionHandler(actionName string, action *types.Action, apiContext *types.APIContext) error {
+	if actionName != "listquestions" {
```

Comment:
Can we also add a check to see if caller of this action has access to this revision using access.ByID

Reply:
This is a collectionAction and not an action on a particular revision. So I think the access check is not applicable here.